timesketch/lib/aggregators/feed.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feed aggregations."""
```

Comment:
Could you add a short description on how to use this aggregation? Seeing it is a bit special

Reply:
saying it's "feed aggregations" is not enough for uyou?

done